source/core/index.ts

```diff
@@ -1358,6 +1361,7 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 			Object.assign(options, urlToOptions(url));
 
 			// `http-cache-semantics` checks this
+			// @ts-ignore
```

Comment:
And here?

Reply:
Some weird TS error, but seems it was fixed in later TS patch versions as I cannot reproduce it now.